llvm/lib/MC/MCCASObjectV1.cpp

```diff
@@ -64,6 +65,14 @@ cl::opt<RelEncodeLoc> RelocLocation(
                clEnumVal(CompileUnit, "In compile unit")),
     cl::init(Atom));
 
+struct CUInfo {
+  size_t CUSize;
+  uint32_t AbbrevOffset;
+};
+static Expected<CUInfo>
+getAndSetDebugAbbrevOffsetAndSkip(MutableArrayRef<char> CUData,
+                                  support::endianness Endian,
+                                  uint32_t NewOffset = 0);
```

Comment:
This forward declaration is here just so I don't have to move the entire function up the file. I'll do this in a separate NFC patch